src/main/java/internet/shop/dao/impl/CarDaoImpl.java

```diff
@@ -1,54 +1,287 @@
+                PreparedStatement preparedStatementOfCarsDrivers = connection
+                        .prepareStatement(queryForCarsDrivers)) {
+            Iterator<Driver> iterator = car.getDrivers().iterator();
+            while (iterator.hasNext()) {
+                preparedStatementOfCarsDrivers.setLong(1, iterator.next().getId());
+                preparedStatementOfCarsDrivers.setLong(2, car.getId());
+                preparedStatementOfCarsDrivers.executeUpdate();
+            }
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't create data "
+                    + car + " into cars_drivers table", exception);
+        }
+    }
+
+    private void removeCarsDrivers(Connection connection, Car car) {
+        String queryRemove = "delete from cars_drivers where car_id = ?;";
+
+        try (PreparedStatement removeStmt = connection
+                .prepareStatement(queryRemove)) {
+            removeStmt.setLong(1, car.getId());
+            removeStmt.executeUpdate();
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't remove data of "
+                    + car + " into cars_drivers", exception);
+        }
+    }
+
+    private void insertCarsDrivers(Connection connection, Car car) {
+        String queryInsert = "INSERT INTO cars_drivers (driver_id, car_id) VALUES (?, ?);";
+
+        try (PreparedStatement insertStmt = connection
+                     .prepareStatement(queryInsert)) {
+            Iterator<Driver> iterator = car.getDrivers().iterator();
+            while (iterator.hasNext()) {
+                insertStmt.setLong(1, iterator.next().getId());
+                insertStmt.setLong(2, car.getId());
+            }
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't insert data of "
+                    + car + " into cars_drivers", exception);
+        }
+    }
+
+    private Car createCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("c_id", Long.class));
+        car.setModel(resultSet.getObject("c_m", String.class));
+        return car;
+    }
+
+    private Manufacturer createManufacturer(ResultSet resultSet) throws SQLException {
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("m_id", Long.class));
+        manufacturer.setName(resultSet.getObject("m_n", String.class));
+        manufacturer.setCountry(resultSet.getObject("m_n", String.class));
+        return manufacturer;
+    }
+
+    private Driver creteDriver(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("d_id", Long.class));
+        driver.setName(resultSet.getObject("d_n", String.class));
+        driver.setLicenceNumber(resultSet.getObject("d_ln", String.class));
```

Comment:
let's have more clear and informative column names in our ResultSets

Reply:
done